dspace-api/src/main/java/org/dspace/content/service/ItemService.java

```diff
@@ -147,6 +147,18 @@
     public Iterator<Item> findAllByCollection(Context context, Collection collection) throws SQLException;
 
     /**
+     * Get all the items in this collection. The order is indeterminate.
+     *
```

Comment:
Could you please add a note how this is different from findByCollection? I assume one contains private and withdrawn items, the other doesn't?

Reply:
@pnbecker , I updated the javadocs for the new methods as well as the legacy methods to clarify which set return each group of items.